common/src/python/outputs/errors.py

```diff
@@ -244,3 +244,22 @@ def errors(self) -> List[Dict[str, Any]]:
           List of serialized FileError objects
         """
         return self.__errors
+
+
+def legacy_naccid_error(value: str,

```

Comment:
we should just use standard logging for these errors, so I don't think you need this